src/app/auth/components/authentication/authentication.component.ts

```diff
@@ -706,31 +707,7 @@ getCookie(key: string){
                 'true'
               );
             }
-
-            if (response.visitedTwitter) {
-              this.tokenStorageService.setSecureWallet(
-                'visited-twitter',
-                'true'
-              );
-            }
-            if (response.visitedLinkedIn) {
-              this.tokenStorageService.setSecureWallet(
-                'visited-linkedin',
-                'true'
-              );
-            }
```

Comment:
t would be beneficial if you could provide some additional information about the changes made.